<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblId">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblIdValue">
     <property name="text">
      <string>idValue</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAnimationName">
     <property name="text">
      <string>Animation name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblAnimationNameValue">
     <property name="text">
      <string>animationNameValue</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPlaybackSpeed">
     <property name="text">
      <string>Playback speed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblPlaybackValue">
     <property name="text">
      <string>playbackSpeedValue</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblIsEnterState">
     <property name="text">
      <string>Is Enter state</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>is Orphan</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxEnterState">
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxOrphan">
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
